appengine/go11x/helloworld/app.yaml

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: go111
+runtime: go112 # replace with go111 for Go 1.11
```

Comment:
I don't think we need the comment for saying how to do it with Go 1.11?

Reply:
Both runtimes are supported and this sample is used in both docs, so I do think we need it.
e.g.
https://cloud.google.com/appengine/docs/standard/go112/building-app/#creating_the_appyaml_file
https://cloud.google.com/appengine/docs/standard/go111/building-app/#creating_the_appyaml_file